### PR TITLE
revision agricultural demand and production

### PR DESCRIPTION
**Pull request by flohump**

### CODE REVIEW

File: definitions/variable/agriculture/agriculture.yaml [hunk 1]
```diff
@@ -9,29 +9,69 @@
...
     definition: Demand for food crops
     unit: million t DM/yr
     agmip: Domestic use (CONS) - ???
     tier: 3
+- Agricultural Demand|Crops|Food|Cereals:
+    definition: Demand for food crops from cereals (maize, rice, temperate cereals and tropical cereals)
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Food|Oil crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:23]:
```suggestion
- Agricultural Demand|Crops|Food|Oil Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 2]
```diff
@@ -9,29 +9,69 @@
...
+    definition: Demand for food crops from oil crops (Cotton seed, Groundnuts, Oilpalms, Other oil crops incl rapeseed, Soybean, Sunflower)
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Food|Sugar crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:28]:
```suggestion
- Agricultural Demand|Crops|Food|Sugar Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 3]
```diff
@@ -9,29 +9,69 @@
...
+    definition: Demand for food crops from sugar crops (Sugar beet, Sugar cane)
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Food|Other crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:33]:
```suggestion
- Agricultural Demand|Crops|Food|Other Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 4]
```diff
@@ -9,29 +9,69 @@
...
+    definition: Demand for food crops from other crops (Fruits Vegetables Nuts, Potatoes, Pulses, Tropical roots)
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:34]:
```suggestion
    definition: Demand for food crops from other crops (fruits, vegetables, nuts, potatoes, pulses, tropical roots)
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 5]
```diff
@@ -9,29 +9,69 @@
...
     definition: Demand for feed crops
     unit: million t DM/yr
     agmip: Domestic use (CONS) - ???
     tier: 3
+- Agricultural Demand|Crops|Feed|Cereals:
+    definition: Demand for feed crops from cereals (maize, rice, temperate cereals and tropical cereals)
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Feed|Oil crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:48]:
```suggestion
- Agricultural Demand|Crops|Feed|Oil Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 6]
```diff
@@ -9,29 +9,69 @@
...
+    definition: Demand for feed crops from oil crops (Cotton seed, Groundnuts, Oilpalms, Other oil crops incl rapeseed, Soybean, Sunflower)
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:49]:
```suggestion
    definition: Demand for feed crops from oil crops (cotton seed, groundnuts, oilpalms, other oil crops incl. rapeseed, soybean, sunflower)
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 7]
```diff
@@ -9,29 +9,69 @@
...
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Feed|Sugar crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:53]:
```suggestion
- Agricultural Demand|Crops|Feed|Sugar Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 8]
```diff
@@ -9,29 +9,69 @@
...
+    definition: Demand for feed crops from sugar crops (Sugar beet, Sugar cane)
+    unit: million t DM/yr
+    agmip: Domestic use (CONS) - ???
+    tier: 4
+- Agricultural Demand|Crops|Feed|Other crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:58]:
```suggestion
- Agricultural Demand|Crops|Feed|Other Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 9]
```diff
@@ -98,31 +153,46 @@
     unit: million t DM/yr
     agmip: Production (PROD) - ???
     tier: 3
-- Agricultural Production|Crops|Sugarcrops:
+- Agricultural Production|Crops|Oil crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:156]:
```suggestion
- Agricultural Production|Crops|Oil Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 10]
```diff
@@ -98,31 +153,46 @@
...
+    definition: Production of crops
+    unit: million t DM/yr
+    agmip: Production (PROD) - ???
+    tier: 3
+- Agricultural Production|Crops|Sugar crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:161]:
```suggestion
- Agricultural Production|Crops|Sugar Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 11]
```diff
@@ -98,31 +153,46 @@
...
     definition: Production of crops
     unit: million t DM/yr
     agmip: Production (PROD) - Sugarcrops (SGC)
     tier: 3
-- Agricultural Production|Crops|Oilcrops:
+- Agricultural Production|Crops|Other crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:166]:
```suggestion
- Agricultural Production|Crops|Other Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 12]
```diff
@@ -98,31 +153,46 @@
     unit: million t DM/yr
     agmip: Production (PROD) - ???
     tier: 3
-- Agricultural Production|Crops|Sugarcrops:
+- Agricultural Production|Crops|Oil crops:
+    definition: Production of crops
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:157]:
```suggestion
    definition: Production of oil crops
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 13]
```diff
@@ -98,31 +153,46 @@
...
+    agmip: Production (PROD) - ???
+    tier: 3
+- Agricultural Production|Crops|Sugar crops:
     definition: Production of crops
     unit: million t DM/yr
     agmip: Production (PROD) - Sugarcrops (SGC)
     tier: 3
-- Agricultural Production|Crops|Oilcrops:
+- Agricultural Production|Crops|Other crops:
     definition: Production of crops
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:167]: This must be updated in line with the variable name.

File: definitions/variable/agriculture/agriculture.yaml [hunk 14]
```diff
@@ -98,31 +153,46 @@
...
     unit: million t DM/yr
     agmip: Production (PROD) - ???
     tier: 3
-- Agricultural Production|Crops|Energy Crops:
+- Agricultural Production|Crops|Energy crops:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:171]:
```suggestion
- Agricultural Production|Crops|Energy Crops:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 15]
```diff
@@ -98,31 +153,46 @@
...
-    agmip: Production (PROD) - ???
-    tier: 2
 - Agricultural Production|Livestock:
-    definition: Production of crops
+    definition: Production of livestock products
     unit: million t DM/yr
     agmip: Production (PROD) - Livestock (LSP)
     tier: 2
+- Agricultural Production|Livestock|Dairy:
+    definition: Production of livestock products
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:182]:
```suggestion
    definition: Production of dairy livestock products
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 16]
```diff
@@ -98,31 +153,46 @@
...
+    unit: million t DM/yr
+    agmip: Production (PROD) - Livestock (LSP)
+    tier: 3
+- Agricultural Production|Livestock|Ruminant:
+    definition: Production of livestock products
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:187]:
```suggestion
    definition: Production of livestock products from ruminants
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 17]
```diff
@@ -98,31 +153,46 @@
...
+    unit: million t DM/yr
+    agmip: Production (PROD) - Livestock (LSP)
+    tier: 3
+- Agricultural Production|Livestock|Non-ruminant:
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:191]:
```suggestion
- Agricultural Production|Livestock|Non-Ruminant:
```

File: definitions/variable/agriculture/agriculture.yaml [hunk 18]
```diff
@@ -98,31 +153,46 @@
...
+    definition: Production of livestock products
```
Comment (danielhuppmann) [inline definitions/variable/agriculture/agriculture.yaml:192]: Please add an explanation what a non-ruminant is for an energy person like me...

Comment (flohump) [reply definitions/variable/agriculture/agriculture.yaml:192]: sure, I can copy-paste this from the demand definition

**Review by danielhuppmann — APPROVED** [review]
Nice work, thank you!

**Pull request merged**